server/client.go

```diff
@@ -252,6 +254,18 @@ type AutoCompleteResult struct {
 	Results []Result `json:"results"`
 }
 
+type Name struct {
+	Name string `json:"name"`
+}
+
+type Group struct {
+	Items []Name `json:"items"`
```

Comment:
why you have named the struct as Item 
```suggestion
	Items []Item `json:"items"`
```